gatheringRequirements/interviews/text.md

```diff
@@ -6,7 +6,7 @@
 
 <div id="body">
 
-**Interviewing stakeholders and <trigger for="pop:interviews-domainExpert">domain experts</trigger> can produce useful information that project requirements.**
+**Interviewing stakeholders and <trigger for="pop:interviews-domainExpert">domain experts</trigger> can produce useful information about project requirements.**
```

Comment:
Not too sure whether `project` is being used here as a verb (i.e. projecting requirements as in forecasting requirements) or `project requirements` is meant to be a noun. If it's the latter, I will revert this change.